libs/execution_context/ebpf_core.c

```diff
@@ -1750,6 +1755,34 @@ _ebpf_core_get_pid_tgid()
     return ((uint64_t)ebpf_platform_process_id() << 32) | ebpf_platform_thread_id();
 }
 
+_Success_(return == 0) static uint32_t
+    _ebpf_core_get_current_logon_id(_In_ const void* ctx, _Out_ uint64_t* logon_id, int size)
+{
+    UNREFERENCED_PARAMETER(ctx);
+    UNREFERENCED_PARAMETER(size);
```

Comment:
this is unsafe.  The verifier will only guarantee that there are 'size' bytes for login_id.
So if you pass it a buffer of 1 byte in length, this implementation will happily write past end of buffer.

Reply:
Updated helper function definition. Removed `size` parameter.